libindy_vdr/src/pool/builder.rs

```diff
@@ -15,16 +17,22 @@ pub struct PoolBuilder {
     transactions: PoolTransactions,
     node_weights: Option<HashMap<String, f32>>,
     refreshed: bool,
+    cache: Option<Cache<String, (String, RequestResultMeta)>>,
 }
 
 impl PoolBuilder {
     /// Create a new `PoolBuilder` instance.
-    pub fn new(config: PoolConfig, transactions: PoolTransactions) -> Self {
+    pub fn new(
+        config: PoolConfig,
+        transactions: PoolTransactions,
+        cache: Option<Cache<String, (String, RequestResultMeta)>>,
```

Comment:
It's confusing to me how PoolBuilder ignores the cache parameter unless it's used to create a PoolRunner, maybe that should just be a parameter of the into_runner method?

Reply:
ya that's a better idea, done